src/css/custom.css

```diff
@@ -181,6 +181,10 @@ dl {
     fill: currentColor;
 }
 
+code {
+  white-space : pre-wrap;
```

Comment:
This is a global change. Are we certain that nothing breaks? I'll pull this in tomorrow and check our config files, **but why are we doing this?**

Reply:
@s-santillan so when you copy one of the code snippets (source code is here https://github.com/returntocorp/semgrep-docs/pull/669/files#diff-db99c4d68b8ef6118fce1fddd68ac32a10a9d1f5f03950e4f223921e152ff6f1R16-R18) it breaks in the terminal where you paste it unless we render it with the soft wrap. Feel free to remove this from the PR if we are breaking something.